MvvmCross/Windows/Uwp/Views/MvxWindowsMultiRegionViewPresenter.cs

```diff
@@ -70,14 +70,14 @@ public override void Close(IMvxViewModel viewModel)
             }
         }
 
-        private static Type GetViewType(MvxViewModelRequest request)
+        protected static Type GetViewType(MvxViewModelRequest request)
         {
             var viewFinder = Mvx.Resolve<IMvxViewsContainer>();
             return viewFinder.GetViewType(request.ViewModelType);
         }
 
         // Implementation from: http://stackoverflow.com/a/1759923/80186
-        internal static T FindChild<T>(DependencyObject reference, string childName) where T : DependencyObject
+        protected internal static T FindChild<T>(DependencyObject reference, string childName) where T : DependencyObject
```

Comment:
This won't break things and is a pretty frequent thing to implement in XAML based apps. Whether or not it should be visible or not it not a huge deal. Will of course clash with implementations with same signature.